processing/src/main/java/io/druid/segment/incremental/IncrementalIndexSchema.java

```diff
@@ -67,19 +70,26 @@ public DimensionsSpec getDimensionsSpec()
     return metrics;
   }
 
+  public boolean isRollup()
+  {
+    return rollup;
+  }
+
   public static class Builder
   {
     private long minTimestamp;
     private QueryGranularity gran;
     private DimensionsSpec dimensionsSpec;
     private AggregatorFactory[] metrics;
+    private boolean rollup;
 
     public Builder()
     {
       this.minTimestamp = 0L;
       this.gran = QueryGranularities.NONE;
       this.dimensionsSpec = new DimensionsSpec(null, null, null);
       this.metrics = new AggregatorFactory[]{};
+      this.rollup = true;
```

Comment:
better to keep rollup=true here, because it will affect lots of UT classes if set rollup to false by default.
If we change the default druid rollup behavior to false in future, we only have to modify the DataSchema.java to set rollup=false if rollup option is not set, and leave these intermediate classes unchanged.